charts/alfresco-sync-service/values.yaml

```diff
@@ -54,83 +54,79 @@ readinessProbe:
+      host: REPO_HOST
+      port: REPO_PORT
+      scheme: REPO_SCHEME
 messageBroker:
   # -- A failover URI formatted string, see:
   # https://activemq.apache.org/failover-transport-reference
   url: null
   # -- A name that will be used as a base to get broker connection details
   nameOverride: activemq
   # -- Username to authenticate as.
-  user: null
+  username: null
   # -- Credential to use to authenticate to the broker.
   password: null
-  # -- An existing k8s secret with broker details (preferred over using values)
-  existingSecretName: null
-# -- Defines properties required by sync service for connecting to the database
-# If you set database.external to true you will have to setup the JDBC driver,
-# user, password and JdbcUrl as `driver`, `user`, `password` & `url` subelements
-# of `database`. Also make sure that the container has the db driver
-postgresql:
-  # -- Toggle PostgreSQL chart dependency
-  # see [PostgreSQL Bitnami charts
-  # documentation](https://github.com/bitnami/charts/tree/main/bitnami/postgresql))
-  enabled: false
-  nameOverride: postgresql-syncservice
-  auth:
-    enablePostgresUser: false
-    username: alfresco
-    password: admin
-    database: alfrescosync
-  primary:
-    extendedConfiguration: |
-      shared_buffers = 256MB
-      max_connections = 100
-      wal_level = minimal
-      max_wal_senders = 0
-      max_replication_slots = 0
-      effective_cache_size = 1024GB
-      log_min_messages = LOG
-  resources:
-    limits:
-      cpu: "2"
-      memory: "2Gi"
+  existingConfigMap:
+    # -- Name of a pre-existing configmap containing message broker details
+    name: null
+    keys:
+      # -- configmap key where to find the URL of the message broker
+      url: BROKER_URL
+  existingSecret:
+    # -- Name of a pre-existing secret containing message broker credentials
+    name: null
+    keys:
+      # -- Key within the secret holding the message broker username
+      username: BROKER_USERNAME
+      # -- Key within the secret holding the message broker password
+      password: BROKER_PASSWORD
 database:
   # -- The JDBC Driver to connect to the DB.
   # If different from the default make sure your container image ships it.
   driver: org.postgresql.Driver
   # -- JDBC url to connect to the external DB
   url: null
   # -- JDBC username to use to connect to the DB
-  user: alfresco
+  username: null
   # -- JDBC password to use to connect to the DB
-  password: admin
-  # -- An existing kubernetes secret with DB info (prefered over using values)
-  existingSecretName: null
+  password: null
+  existingConfigMap:
+    name: null
+    keys:
+      # -- configmap key where to find the URL of the database
+      url: DATABASE_URL
+      # -- configmap key where to find the JDBC driver class to use.
+      # The configmap may leverage the alfresco-repository.db.cm named
+      # template to auto-generate it from the sole url parameter.
+      driver: DATABASE_DRIVER
```

Comment:
not sure it make sense to have driver here but I understand is correlated to url so if it change then you may have to provide a different driver